### ISSUE TITLE
Doric 0.9.0 broken with cyclic load dependencies

### ISSUE THREAD

**Issue by jaydeesimon** [babashka/babashka#741]
## Version

v0.2.11

## Platform

macOS Catalina 10.15.7

## Problem

My Babashka scripts that use Doric 0.9.0 break with Babashka v0.2.11. The error is related to cyclic load dependencies. Although this may be by design because of https://github.com/borkdude/sci/issues/531.

## Reproduction

```
Babashka v0.2.11 REPL.
Use :repl/quit or :repl/exit to quit the REPL.
Clojure rocks, Bash reaches.

user=> (require '[babashka.deps :as deps])
nil
user=> (deps/add-deps '{:deps {doric/doric {:mvn/version "0.9.0"}}})
nil
user=> (require '[doric.core :as d])
nil
user=> (d/table [:a :b] [{:a 1 :b 2}])
Cyclic load dependency: [ doric.core ]->doric.org->[ doric.core ] [at doric/org.clj:4:10]
```

## Expected

```
Babashka v0.2.10 REPL.
Use :repl/quit or :repl/exit to quit the REPL.
Clojure rocks, Bash reaches.

user=> (require '[babashka.deps :as deps])
nil
user=> (deps/add-deps '{:deps {doric/doric {:mvn/version "0.9.0"}}})
nil
user=> (require '[doric.core :as d])
nil
user=> (d/table [:a :b] [{:a 1 :b 2}])
"|---+---|\n| A | B |\n|---+---|\n| 1 | 2 |\n|---+---|"
```

or with Clojure 1.10.2

```
$ cat deps.edn 
{:deps
 {doric/doric {:mvn/version "0.9.0"}}}

$ clj         
Clojure 1.10.2
user=> (require '[doric.core :as d])
nil
user=> (d/table [:a :b] [{:a 1 :b 2}])
"|---+---|\n| A | B |\n|---+---|\n| 1 | 2 |\n|---+---|"
```

**Sponsor**

**Comment by borkdude** [comment]
@jaydeesimon I'm sorry for the breakage. Babashka should not be any more strict than clojure, the cyclic dependency is supposed to be helpful, not overly strict. I'll see why this happens and will try to fix.

**Comment by borkdude** [comment]
Fixed, releasing 0.2.12 in a few moments

**Comment by jaydeesimon** [comment]
Thanks @borkdude! I had considered that the issue was with the library but when I tried it with Clojure 1.10.2 and saw that there was no issue with it, I figured you’d want to know about it since I remember reading that the ns cycle detection was based on Clojure’s.

Thanks again for the quick response. Your time is much appreciated.
